### ISSUE TITLE
Rethink Angular as the front-end framework

### ISSUE THREAD

**Issue by gouthamve** [LambdaIITH/Dashboard-Old#5]
I think Angular is overkill for this simple thing. Will slow us down.

**Comment by CrazyByDefault** [comment]
We understand your concern, but we don't view this project as a 1-month hackathon. We want to take this project forward and implement it as soon as possible.

**Comment by gouthamve** [comment]
Yes. But Angular if you ask me is not something to be taken lightly!
